android/assets/jsons/Civ V - Gods & Kings/TileImprovements.json

```diff
@@ -5,100 +5,90 @@
 		"terrainsCanBeBuiltOn": ["Plains","Grassland","Desert","Flood plains"],
 		"food": 1,
 		"turnsToBuild": 7,
-        "pillageGold": 18,
 		"techRequired": "Agriculture",
 		"uniques": ["Can also be built on tiles adjacent to fresh water",
 			"[+1 Food] from [Fresh water] tiles <after discovering [Civil Service]>",
-			"[+1 Food] from [non-fresh water] tiles <after discovering [Fertilizer]>"],
+			"[+1 Food] from [non-fresh water] tiles <after discovering [Fertilizer]>",
+            "Yields on average [+18 Gold] when pillaged"],
 		"shortcutKey": "F"
 	},
 	{
 		"name": "Lumber mill",
 		"terrainsCanBeBuiltOn": ["Forest"],
 		"production": 1,
 		"turnsToBuild": 7,
-        "pillageGold": 10,
 		"techRequired": "Construction",
-		"uniques": ["[+1 Production] <after discovering [Scientific Theory]>"],
+		"uniques": ["[+1 Production] <after discovering [Scientific Theory]>", "Yields on average [+10 Gold] when pillaged"],
 		"shortcutKey": "L"
 	},
 	{
 		"name": "Mine",
 		"terrainsCanBeBuiltOn": ["Hill"],
 		"production": 1,
 		"turnsToBuild": 7,
-        "pillageGold": 20,
 		"techRequired": "Mining",
-		"uniques": ["[+1 Production] <after discovering [Chemistry]>"],
+		"uniques": ["[+1 Production] <after discovering [Chemistry]>", "Yields on average [+20 Gold] when pillaged"],
 		"shortcutKey": "M"
 	},
 	{
 		"name": "Trading post",
 		"terrainsCanBeBuiltOn": ["Plains","Grassland","Desert","Hill","Tundra","Forest","Jungle","Flood plains"],
 		"gold": 1,
 		"turnsToBuild": 7,
-        "pillageGold": 20,
 		"techRequired": "Guilds",
-		"uniques": ["[+1 Gold] <after discovering [Economics]>"],
+		"uniques": ["[+1 Gold] <after discovering [Economics]>", "Yields on average [+20 Gold] when pillaged"],
```

Comment:
Sorry, not sure this is the best possible wording. And as you know changing opinions after a release gets tricky... "Yields aproximately...", "Pillaging yields [20 Gold] on average"? I'm no native brit or yankee, but I hesitated an instant 'does the on refer to  yields or to average?'... "You can loot an average of X when pillaging"? Not sure. OK, I'll not veto any choice, just do what feels best.

Reply:
It's definitely difficult trying to get the point across in a succinct way that this unique type has randomization in (a non-uniform distribution too). I'll play around with it a bit but I like your suggestion of renaming the uniques to "Pillaging yields []..." since that also puts "pillaging" at the beginning of the line and makes it immediately clear to the reader what the line is talking about when it starts listing a bunch of stats.

Reply:
Updated the unique text to be more clear.